can-bus/json/PDM/PDM_rx.json

```
@@ -1,6 +1,7 @@
 {
     "messages": [
         "BMS_Vitals",
-        "BMS_Contactors"
+        "BMS_Contactors",
+        "Debug_CanModes"
     ]
 }
```

nit: missing blank line
```suggestion
}

```